src/Providers/StoreFileProvider.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Providers;
+
+use Blumilk\Meetup\Core\Contracts\StoreFile;
+use Blumilk\Meetup\Core\Services\StoreFileService as Service;
```

Comment:
Because the code is short, concise and you don't need to break it down

Reply:
I still don't get it. In case of any other service bindng, your approach will be at least ambiguous. I would stay with imported real class name `StoreFileService` and use it in binder. 

Btw. you can use `$this->app->bind()` instead of helper `app()`.